### ISSUE TITLE
Remove Tether

### ISSUE THREAD

**Issue by sir-dunxalot** [sir-dunxalot/ember-tooltips#221]
Tether has become more of a detriment than an aid to this library. Furthermore, the developers of Tether have stopped actively maintaining the project. As such, we should remove Tether as a dependency and find a more reliable way to position tooltips.

I have begun work to remove Tether and replace it with popper.js/tooltip.js [on this branch](https://github.com/sir-dunxalot/ember-tooltips/tree/remove-tether-dependency).

**Comment by kybishop** [comment]
@sir-dunxalot you might want to consider looking into https://github.com/kybishop/ember-popper. Also, It's my understanding that tooltip.js is more of a showcase for popper.js than a full-fledged effort, but that's not to say it won't do what is needed!

For comparison, ember-attacher was developed as an ember-first impl of tippy.js, which uses popper.js for positioning. Might be more worth basing the next version of ember-tooltips off tippy.js than tooltip.js.

**Comment by sir-dunxalot** [comment]
Thanks for the tips @kybishop (no pun intended)! I had a read through your ember-popper library a little while back - good stuff!

The latest iteration of a 'tetherless' implementation of this library uses `popper.js` and `ember-wormhole` alongside the existing events for hiding and showing tooltips. Luckily, event management has always been a strong part of this library and the positioning has been a weak part so hopefully hat's a good solution.

I originally started with `ember-popper` but I ended up deferring to the core popper library. I forget why but let me try and recall the reasons later - perhaps we can adapt `ember-popper` to accommodate for whatever issues I ran into and then it can be incorporated into `ember-tooltips`.
